aisim/atoms.py

```diff
@@ -35,10 +35,13 @@ class AtomicEnsemble():
         the atoms in an atomic ensemble
     """
 
-    def __init__(self, phase_space_vectors, state_vectors=[1, 0], time=0, weights=None):
+    def __init__(self, phase_space_vectors, state_kets=[1, 0], time=0, weights=None):
         assert phase_space_vectors.shape[1] == 6
         self.phase_space_vectors = phase_space_vectors
-        self.state_vectors = state_vectors
+        if isinstance(state_kets, list):
+            self.state_kets = np.array([state_kets]).T
+        else:
+            self.state_kets = state_kets

```

Comment:
This is all alrady done by the setter of the state_ket property!

Reply:
yes. removed it.